deploy/examples/create-external-cluster-resources.py

```diff
@@ -515,6 +527,8 @@ def create_cephCSIKeyring_cephFSProvisioner(self):
                         "caps": ["mon", "allow r", "mgr", "allow rw",
                                  "osd", "allow rw tag cephfs metadata=*"],
                         "format": "json"}
+        if self._arg_parser.dry_run:
+            return self.dry_run("ceph " + cmd_json['prefix'] + " " + cmd_json['entity'] + " " + " ".join(cmd_json['caps']))
```

Comment:
can you please explain `" ".join(cmd_json['caps']))` what this symbolizes.

Reply:
this is join dict to str.